v1/transform_spans_test.go

```diff
@@ -139,10 +139,10 @@ func TestOCSpanToProtoSpan_endToEnd(t *testing.T) {
 		},
 		Attributes: &tracepb.Span_Attributes{
 			AttributeMap: map[string]*tracepb.AttributeValue{
-				"cache_hit":  &tracepb.AttributeValue{Value: &tracepb.AttributeValue_BoolValue{BoolValue: true}},
```

Comment:
@bogdandrutu could you please ensure we use the plain `gofmt`?

Reply:
That's just a human stylistic reformat(removing redunancy) but will turn spurious and is unrelated to adding a Travis config :)

Reply:
Adding travis config fails because code is not formatted correctly. What can I do?

We are doing this in travis:
  - if [ -n "$(gofmt -s -l $GO_FILES)" ]; then echo "gofmt the following files:"; gofmt -s -l $GO_FILES; exit 1; fi
  - go vet ./...

So I need to pass tests and checks before I can submit it (the config is the same as in the main opencensus-go repo which I think is the right thing to do).

Reply:
Remember that adding a config triggers the tests/checks to run so what should I do?

Reply:
Gotcha,`gofmt -s -w` is new in this PR, but anyways in that case for sure, let's go with all the changes, thanks for making them.